CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,126 @@
+/*:
+ 
+ ## Github Client Example.
+
+ This is a demonstration of using Cleanse and NSURLSession and friends to configure a github client that
+ will list a User's projects
+*/
+
+import Cleanse
+
+import XCPlayground
+
+
+//: Since we will be doing an asynchronous call, we need to make it so the execution doesn't stop immediately
+XCPlaygroundPage.currentPage.needsIndefiniteExecution = true
+
+//: This is the base URL for github
```

Comment:
For a playground it makes nice formatting

https://developer.apple.com/library/ios/documentation/Xcode/Reference/xcode_markup_formatting_ref/Headings.html#//apple_ref/doc/uid/TP40016497-CH8-SW1

<img width="1405" alt="githubclientexample_xcplaygroundpage" src="https://cloud.githubusercontent.com/assets/21435/16051407/867a92ac-3214-11e6-8247-8fab513ccbe9.png">